collections/crop-biomass.yaml

```diff
@@ -0,0 +1,39 @@
+Name: Crop Biomass
+Description: |
+  [Crop Biomass](https://www.planet.com/products/planetary-variables/) provides the ability to track and respond to the highly dynamic nature of biomass in agricultural fields throughout the season. 
+  This enables better-informed decisions, input optimization, interventions, and harvest planning. 
+  This is a relative measure of biomass, so each pixel value has a value of 0 (low biomass) to 1 (high biomass). 
+  The product is provided daily and every day, the user gets a new raster with the data from the most updated direct observation from the day before.
+AdditionalInfoExternal:
+    Title: Open Data for Crop Biomass
+    Path: crop-biomass/open-data.md
+Documentation: "[here](https://docs.sentinel-hub.com/api/latest/data/planetary-variables/crop-biomass/)"
+Image: crop-biomass/crop-biomass.png
+Resolution: 10m
+GeographicalCoverage: Global; Open data in [limited areas](crop-biomass/open-data.html); 
```

Comment:
The root url is  `https://collections.sentinel-hub.com/crop-biomass/`. (should have mentioned this already)

Therefore, the link `(crop-biomass/open-data.html)`  is one level too high  

You just need  `(open-data.html)`.  That's it. 

You will see that if you do a local  build , see instructions on how to build the site on local [Link](https://github.com/sentinel-hub/public-collections/blob/main/CONTRIBUTING.md#how-to-build-sentinel-hub-public-collections-website-locally)